averages.js

```diff
@@ -0,0 +1,54 @@
+function total(nums){
+  numTotal = nums.reduce( function(sum, num){
+    return sum + num;
+  }, 0);
+  return numTotal
+}
+
+function mean(nums) {
+  return total(nums) / nums.length
+}
+
+function sortNums(nums) {
+  sorted = nums.sort(function (a, b) {
+    return a - b;
+  }); 
+  return sorted;
+}
+
+function median(nums) {
+  var sortedNums = sortNums(nums)
+  if (sortedNums.length%2){
+    var i = Math.floor(sortedNums.length/2)
```

Comment:
I would recommend using a variable with some semantic meaning here for readability